adss-rs/Cargo.toml

```diff
@@ -10,8 +10,8 @@ edition = "2018"
 [dependencies]
 strobe-rs = "0.8.1"
 strobe-rng = { path = "../strobe-rng" }
-rand = "0.8.5"
 zeroize = "1.5.5"
+rand_core = { version = "0.6.3", features = ["getrandom"] }
```

Comment:
@evq updater their PR here https://github.com/brave/sta-rs/pull/240/files, but I think missed `getrandom`

Reply:
Ah, they just went to the same code through a different import path. Does the same thing, but this PR includes less code in the build input, I think.